ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ip-routing.j2

```diff
@@ -7,13 +7,26 @@
 
 | VRF | Routing Enabled |
 | --- | --------------- |
-| default | {{ ip_routing | arista.avd.default(false) }} |
-{%     for vrf in vrfs | arista.avd.natural_sort('name') %}
-{%         if vrf.ip_routing is arista.avd.defined(true) %}
-| {{ vrf.name }} | true |
-{%         elif vrf.ip_routing is arista.avd.defined(false) %}
-| {{ vrf.name }} | false |
+{%     set default_ip_routing = ip_routing | arista.avd.default(false) %}
+{%     set default_ip_routing_ipv6_interfaces = ip_routing_ipv6_interfaces | arista.avd.default(false) %}
+{# Handle the case where VRF default exists #}
+{%     set vrf_default = vrfs | arista.avd.default([]) | selectattr("name", "arista.avd.defined", "default") | first %}
+{%     if vrf_default is arista.avd.defined %}
+{%         set default_ip_routing = (vrf_default.ip_routing or default_ip_routing) %}
+{%         if default_ip_routing_ipv6_interfaces or (vrf_default.ip_routing_ipv6_interfaces | arista.avd.default(false)) %}
+{%             set default_ip_routing = "True (ipv6 interfaces)" %}
 {%         endif %}
+{# No VRF "default" defined - checking if default_ip_routing_ipv6_interfaces is True #}
```

Comment:
I think this could be simplified by just setting `default_vrf_xxx` vars from vrf with defaults and then do the or check afterwards.

Reply:
good point - the only thing is that I am playing with booleans and strings but I was not completely pleased with the logic so will rearrange- thanks for feedback